lib/brainstem/api_docs/formatters/markdown/helper.rb

```diff
@@ -69,6 +69,15 @@ def md_li(text, indent_level = 0)
           def md_a(text, link)
             "[#{text}](#{link})"
           end
+
+
```

Comment:
👮extra newline?

Reply:
@juanca @ahuth I am try to keep the formatting consistent. Once the documentation generation effort is complete, I will replace the double line breaks as a separate PR

Reply:
Whichever way works with me in the short term. Long term, I would like to have a discussion about adding some linting ruleset for the sake of consistency.